### ISSUE TITLE
Compilation fails at CompletionOptionsImpl.java:19 

### ISSUE THREAD

**Issue by bruno-medeiros** [TypeFox/ls-api#32]
I was trying the latest source, but:

```
$ ./gradlew build
To honour the JVM settings for this build a new JVM will be forked. Please consider using the daemon: https://docs.gradle.org/2.14.1/userguide/gradle_daemon.html.
:io.typefox.lsapi.annotations:generateXtext UP-TO-DATE
:io.typefox.lsapi.annotations:compileJava UP-TO-DATE
:io.typefox.lsapi.annotations:processResources UP-TO-DATE
:io.typefox.lsapi.annotations:classes UP-TO-DATE
:io.typefox.lsapi.annotations:jar
:io.typefox.lsapi:generateXtext
:io.typefox.lsapi:compileJavawarning: [options] bootstrap class path not set in conjunction with -source 1.7
D:\devel\git\ls-api\io.typefox.lsapi\src\main\xtend-gen\io\typefox\lsapi\CompletionOptionsImpl.java:19: error: CompletionOptionsImpl is not abstract and does not override abstract method getResolveProvider() in CompletionOptions
public class CompletionOptionsImpl implements CompletionOptions {
       ^
D:\devel\git\ls-api\io.typefox.lsapi\src\main\xtend-gen\io\typefox\lsapi\CompletionOptionsImpl.java:27: error: getResolveProvider() in CompletionOptionsImpl cannot implement getResolveProvider() in CompletionOptions
  public boolean getResolveProvider() {
                 ^
  return type boolean is not compatible with Boolean
D:\devel\git\ls-api\io.typefox.lsapi\src\main\xtend-gen\io\typefox\lsapi\CompletionOptionsImpl.java:26: error: method does not override or implement a method from a supertype
  @Override
  ^
D:\devel\git\ls-api\io.typefox.lsapi\src\main\xtend-gen\io\typefox\lsapi\SymbolInformationImpl.java:20: error: SymbolInformationImpl is not abstract and does not override abstract method getContainerName() in SymbolInformation
public class SymbolInformationImpl implements SymbolInformation {
       ^
D:\devel\git\ls-api\io.typefox.lsapi\src\main\xtend-gen\io\typefox\lsapi\SymbolInformationImpl.java:72: error: method does not override or implement a method from a supertype
  @Override
  ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
5 errors
1 warning
 FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':io.typefox.lsapi:compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 20.927 secs
```

**Comment by JanKoehnlein** [comment]
Just had the same issue here: The fix is to call 
   git clean -fdx
first. Xtend does not hook in properly into the gradle clean task.

**Comment by oehme** [comment]
See https://github.com/xtext/xtext-gradle-plugin/issues/59

**Comment by bruno-medeiros** [comment]
I invoked `git clean -fdx` once and the compilation works now. 
But, at the moment, do I need to call `git clean -fdx` everytime before a build, in case XTend source has changed?

**Comment by oehme** [comment]
Only if an Xtend file was deleted. I'll try to fix this one as soon as I can, as it is pretty serious.

**Comment by oehme** [comment]
Fixed in 1.0.8, which will be out in a couple minutes.

**Comment by mickaelistria** [comment]
I've built a few times ls-api today without any issue of that kind whereas I got it at the same time as @bruno-medeiros reported it. So it seems to me that this issue is resolved and could be closed.
